diff -up url-2.2.1/Cargo.toml.1~ url-2.2.1/Cargo.toml
--- url-2.2.1/Cargo.toml.1~	2021-03-28 22:48:28.181783817 +0200
+++ url-2.2.1/Cargo.toml	2021-03-28 22:48:53.454701038 +0200
@@ -23,6 +23,7 @@ keywords = ["url", "parser"]
 categories = ["parser-implementations", "web-programming", "encoding"]
 license = "MIT/Apache-2.0"
 repository = "https://github.com/servo/rust-url"
+exclude = ["/.travis.yml", "/appveyor.yml"]
 
 [[bench]]
 name = "parse_url"
